imports/components/@enhancers/security-roles/__tests__/index.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+
+import { Component } from "react";
+import { shallow } from "enzyme";
+import { shallowToJson } from "enzyme-to-json";
+import { SECURITY_ROLES_QUERY, canSee, authorized } from "../";
+
+it("has a valid query", () => {
+  expect(SECURITY_ROLES_QUERY).toMatchSnapshot();
+});
+
+it("is renders a wrapper function", () => {
+  expect(canSee(["Staff"])).toMatchSnapshot();
```

Comment:
You could maybe call this around something like a `div` to show the wrapper works. That, or you could just expect the typeof that call to be a function rather than snapshotting it, since the snapshot it just `[Function]`